src/main/resources/assets/tconstruct/book/materials_and_you/en_us/language.lang

```diff
@@ -5,19 +5,26 @@ bowmaterials=Bow materials
 tools=Tools
 modifiers=Modifiers
 smeltery=Smeltery
-seared_furnace=Seared Furnace
-tinker_tank=Tinker Tank
 tier_one_materials=Tier 1 Materials
 
 repairing=Repairing
 offhand=Offhand Harvesting
-bolt_core=Bolt Crafting
+
+bonus_upgrades=§1§lBonus Upgrades§r
+general=§1§lGeneral§r
+harvest=§1§lHarvest§r
+weapon=§1§lWeapon§r
+slotless=§1§lSlotless§r
+ability=§1§lAbility§r
+further_reading=Further Reading
+terminology=Terminology
+
 
 modifier.effect=Effects:
 tool.properties=Properties:
 
 material.craft_partbuilder=Can be crafted in the Part Builder
-material.craft_casting=Can be cast from 
+material.craft_casting=Can be cast from %s
 
-material.tconstruct.wood.book=How much wood could a woodchuck chuck if a woodchuck could chuck wood?
-material.tconstruct.stone.book=Rock solid
+wood.flavour=How much wood could a woodchuck chuck if a woodchuck could chuck wood?
```

Comment:
I changed the format for flavor text to not ignore mod ID for the material, so as a result you deleted the flavor text in this commit. 

I'd suggest adding flavor text for all 5 materials anyways, since everything else has flavor text.